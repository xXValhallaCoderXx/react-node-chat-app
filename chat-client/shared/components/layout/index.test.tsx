import React, { Fragment } from 'react';
import { shallow } from 'enzyme';
import Layout from './index';

describe('Layout', () => {
  it('Renders correctly', () => {
    const wrapper = shallow(<Layout header={<Fragment>Header</Fragment>} />);

    expect(wrapper).toMatchSnapshot();
    expect(wrapper.exists('#header')).toBe(true);
    expect(wrapper.find('#header').length).toBe(1);
    expect(wrapper.find('#header').text()).toEqual('Header');
    expect(wrapper.exists('#footer')).toBe(false);
  });
});
