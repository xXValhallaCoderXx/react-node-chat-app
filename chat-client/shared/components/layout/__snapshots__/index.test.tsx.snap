// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout Renders correctly 1`] = `
<Container
  className="main"
  fluid={true}
  tag="div"
>
  <div
    id="header"
  >
    Header
  </div>
  <Row
    className="h-100"
    tag="div"
  />
</Container>
`;
